Solo Apprentice Death Events (Beach)

CL0WNTH0UGHTS opened the pull request:
Adds 12 new apprentice death events, all under one specific idea (the apprentice dies from dehydration and overheating.

NOTES
-there isnt a tag for sibling only death events(if there is, I couldnt find it on the doc) 
-some of these might be too graphic, let me know to change them.

Commit:
Solo Apprentice Death Events (Beach)

Adds 12 new apprentice death events, all under one specific idea (the apprentice dies from dehydration and overheating.

NOTES
-there isnt a tag for sibling only death events(if there is, I couldnt find it on the doc)
-some of these might be too graphic, let me know to change them.
-expect more

Commit:
Fix Ascii character

whoooooooooops, fixed a left over smart comma (from when I was organizing stuff)

CL0WNTH0UGHTS committed:
Merge branch 'Thlumyn:development' into development

keyraven commented on resources/dicts/events/death/beach/apprentice.json:
```diff
@@ -1,3 +1,225 @@
+            "other_cat",
+            "other_cat_mentor"
+        ],
+        "death_text": "r_c noticed m_c hadn't shown up for training again, yet after further prodding realized {PRONOUN/m_c/subject} hadn't been at camp for a few days. However, r_c never expected to find m_c like this: dead and dried out, round black beetles crawling in, out, and across {PRONOUN/m_c/poss} once lively pelt.",
```
We should avoid any "gross" descriptions of decay or gore. Community feedback has been negative for such events. 

I suggest removing the descriptions of insects crawling over the corpse. I think "dried out" is okay, but people may bulk at insect imagery. 

Suggested:
"r_c noticed m_c hadn't shown up for training again, yet after further prodding realized {PRONOUN/m_c/subject} hadn't been at camp for a few days. However, r_c never expected to find m_c like this: dead and dried out, their tattered pelt lifeless and dull."

keyraven commented on resources/dicts/events/death/beach/apprentice.json:
```diff
@@ -1,3 +1,225 @@
+         "other_cat",
+         "other_cat_parent"
+        ],
+        "death_text": "r_c is frantic with worry; m_c, {PRONOUN/r_c/poss} child, hadn't been seen around camp, no matter who {PRONOUN/r_c/subject} asked and where {PRONOUN/r_c/subject} searched. It's not until r_c stumbles across m_c's body, withered under the sun and being feasted on by black and orange butterflies, with a despairing sound, that {PRONOUN/r_c/subject} {VERB/r_c/realize/realizes}  ...
```
Similar issue here.  I suggest revising. 

Suggestion: 
"It's not until r_c stumbles across m_c's body, withered under the sun, that, with a despairing sound, {PRONOUN/r_c/subject} {VERB/r_c/realize/realizes} {PRONOUN/r_c/poss} greatest fears have come true."

keyraven commented on resources/dicts/events/death/beach/apprentice.json:
```diff
@@ -1,3 +1,225 @@
+            "other_cat",
+            "other_cat_mentor"
+        ],
+        "death_text": "m_c knew {PRONOUN/m_c/subject} made a mistake, as {PRONOUN/m_c/subject} felt {PRONOUN/m_c/poss} last bits of strength slip away from {PRONOUN/m_c/object} and {PRONOUN/m_c/poss} heart stopped beating. {PRONOUN/m_c/poss/CAP} body only discovered days later, r_c wailing at the shriveled body of {PRONOUN/r_c/poss} apprentice, who succumbed to heat and dehydration, covered in be ...
```
This one borderline, but it may be best to avoid anything that implies insects are eating the body.

keyraven commented on resources/dicts/events/death/beach/apprentice.json:
```diff
@@ -1,3 +1,225 @@
+         "other_cat",
+         "other_cat_parent"
+        ],
+        "death_text": "m_c  never meant to meet StarClan this way, rotting away under the sun after fainting from dehydration, heat exhaustion, and overexertion. r_c tragically discovers {PRONOUN/m_c/poss} body, unable to stop {PRONOUN/r_c/poss} desperate wails at the site of {PRONOUN/r_c/poss} child's decaying corpse, being feasted on by butterflies and other scavengers.",
```
Phrases like "{their} child's decaying corpse" may be too graphic, based on community feedback.  The community has been outspoken, particularly, on descriptions of the decay of young cat's bodies. 

Additionally, there is also insect imagery here.

keyraven commented on resources/dicts/events/death/beach/apprentice.json:
```diff
@@ -1,3 +1,225 @@
+         "other_cat",
+         "other_cat_adult"
+        ],
+        "death_text": "r_c's mournful sounds pierced through c_n's territory as {PRONOUN/r_c/subject} stumble on the lifeless, decaying corpse of m_c. m_chad disappeared from camp days ago, having not returned, but only now did r_c understand why  {PRONOUN/r_c/poss} child didn't come back home; Rose was caught out in the treacherous sun, ill-prepared, succumbing to dehydration and exhaustion.",
```
Typo: There is missing a space between "m_c" and "had" at the beginning of the second sentence.

CL0WNTH0UGHTS committed:
Merge branch 'Thlumyn:development' into development

Commit:
Changes and Grammar to the Events

-fixes any grammar mistakes that were pointed out in the review (Thank you Key!)
-remove any graphic mentions of bugs (due to player consensus aswell as some of them being explicit in hindsight, thank you Key for informing me)
-remove explicit mentions to young cat deaths (Once again, due to player consensus, thank you Key for informing me!)

Commit:
Merge branch 'development' of https://github.com/CL0WNTH0UGHTS/Summoners-Clownthoughts-Death-Events into development

keyraven commented on resources/dicts/events/death/beach/apprentice.json:
```diff
@@ -1,3 +1,225 @@
+         "other_cat",
+         "other_cat_adult"
+        ],
+        "death_text": "r_c's mournful sounds pierced through c_n's territory as {PRONOUN/r_c/subject} stumble on the lifeless, decaying corpse of m_c. m_c had disappeared from camp days ago, having not returned, but only now did r_c understand why  {PRONOUN/r_c/poss} child didn't come back home; Rose was caught out in the treacherous sun, ill-prepared, succumbing to dehydration and exhaustion.",
```
Is "Rose" meant to be m_c?

CL0WNTH0UGHTS replied:
Yes, I apologize for leaving that in there! I normally do names (beyond m_c/r_c) to test grammar and see if anything needs to be changed to comply with gendered pronouns. Thank you for catching that 👍

CL0WNTH0UGHTS committed:
Merge branch 'Thlumyn:development' into development

Commit:
changes an event

Accidently left the placeholder name "Rose" from when I was checking for grammar and pronouns. Whoops D:

keyraven commented:
This is approved, but will be merged after the Halloween release.
